tests/conftest.py

```diff
@@ -515,18 +516,27 @@ def _generate_inputs_pw():
         from aiida_quantumespresso.utils.resources import get_default_options
 
         inputs = {
-            'code': fixture_code('quantumespresso.pw'),
-            'structure': generate_structure(),
-            'kpoints': generate_kpoints_mesh(2),
-            'parameters': Dict(dict={
-                'CONTROL': {
-                    'calculation': 'scf'
-                },
-                'SYSTEM': {
-                    'ecutrho': 240.0,
-                    'ecutwfc': 30.0
+            'code':
+            fixture_code('quantumespresso.pw'),
+            'structure':
+            generate_structure(),
+            'kpoints':
+            generate_kpoints_mesh(2),
+            'parameters':
+            Dict(
+                dict={
+                    'CONTROL': {
+                        'calculation': 'scf'
+                    },
+                    'SYSTEM': {
+                        'ecutrho': 240.0,
+                        'ecutwfc': 30.0
+                    },
+                    'ELECTRONS': {
+                        'electron_maxstep': 60,
+                    }
```

Comment:
What happened here with the auto-formatter? This is crazy...

Reply:
Haha, right, didn't notice. Let's see if it keeps being difficult.

Reply:
Got it to behave! 😁